templates/error_page.inc.php

```diff
@@ -41,7 +41,7 @@
     <div class="navbar navbar-inverse navbar-fixed-top" role="navigation">
         <div class="container">
             <a class="navbar-brand" href="#">
-                <img src="./images/ampache-dark.png" title="<?php echo $t_ampache; ?>" alt="<?php echo $t_ampache; ?>">
```

Comment:
the error page needs to be hardcoded for install/test errors

Reply:
Sounds good, will fix